src/sequence.js

```diff
@@ -30,8 +31,12 @@ var reduceRight = require('./reduceRight');
  *      R.sequence(R.of, Nothing());       //=> [Nothing()]
  */
 module.exports = _curry2(function sequence(of, traversable) {
-  return typeof traversable.sequence === 'function' ?
-    traversable.sequence(of) :
+  if (typeof traversable.sequence === 'function') {
+    return traversable.sequence(of);
+  }
```

Comment:
I'm strongly in favour of not dispatching to `sequence`, which is vestigial.

:thumbsup: / :thumbsdown: ?